src/cliAgent.ts

```
@@ -14,27 +14,35 @@
-  V2ProofProtocol,
-  CredentialsModule,
-  V2CredentialProtocol,
-  ConnectionsModule,
   W3cCredentialsModule,
   KeyDidRegistrar,
   KeyDidResolver,
   CacheModule,
   InMemoryLruCache,
   WebDidResolver,
+  LogLevel,
+  Agent
```

ESLint

[error] Insert `,`

(prettier/prettier)